### ISSUE TITLE
Upgrading BDD Zabbix 

### ISSUE THREAD

**Issue by Setsuneh** [YunoHost-Apps/zabbix_ynh#44]
Hi,

Is it possible to update  Zabbix_ynh? Seems so many problems on fresh Installation (Debian 11, 64bits) : 

`La version de la base de données Zabbix ne correspond pas aux exigences actuelles. Version de votre base de données : 5000000. Version requise : 6000000. Veuillez contacter votre administrateur système.`

**Comment by stepcellwolf** [comment]
is there any way to upgrade to 7.0 LTS version too?

**Comment by stepcellwolf** [comment]
Version 6.4 from branch is not workin it gives error. Any news on this? I will be more than happy to help in upgrading it to more latest version.

**Comment by Thatoo** [comment]
Indeed, upgrading the package to 7.0 or at least 6.4 is required as v5 doesn't support bookworm.
